tests/test_otaclient/test_create_standby/test_delta_gen_e2e.py

```diff
@@ -22,12 +22,13 @@
 import pytest
 import pytest_mock
 
-from ota_metadata.legacy2.metadata import OTAMetadata
+from ota_metadata.file_table.db import FileTableDBHelper
 from otaclient.create_standby.delta_gen import (
     InPlaceDeltaGenFullDiskScan,
     RebuildDeltaGenFullDiskScan,
     RebuildDeltaWithBaseFileTable,
 )
+from otaclient_common.thread_safe_container import ShardedThreadSafeDict
```

Comment:
Is it `ResourcesDigestWithSize`? In my environment, the following warning is displayed.
```
Argument of type "ShardedThreadSafeDict[Unknown, Unknown]" cannot be assigned to parameter "all_resource_digests" of type "ResourcesDigestWithSize" in function "__init__"
  "ShardedThreadSafeDict[Unknown, Unknown]" is not assignable to "ResourcesDigestWithSize"Pylance[reportArgumentType](https://github.com/microsoft/pylance-release/blob/main/docs/diagnostics/reportArgumentType.md)
(variable) _all_digests: ShardedThreadSafeDict[Unknown, Unknown]
```

Reply:
Ah I forgot to fix up the test!
Fixed in [bedcd25](https://github.com/tier4/ota-client/pull/637/commits/bedcd25c8b4e9ffac079afc17f936ebea5513a8d)